sles/src/stack/images/common/sles-stacki.img-patches/opt/stack/bin/stacki-profile.py

```diff
@@ -7,146 +7,131 @@
 # @copyright@
 #
 
-import subprocess
+import json
+import os
 import random
+import re
+import subprocess
+import sys
 import time
-import os
-import json
-
-def get_ipmi_mac():
-	# Get IPMI mac
-	mac = None
-	p = subprocess.Popen(["/usr/bin/ipmitool", "lan","print","1"], stdout=subprocess.PIPE, stderr=subprocess.PIPE)
-	o, e = p.communicate()
 
-	if not o:
-		return None
 
-	for line in o.decode().split('\n'):
-		if line.startswith("MAC Address"):
-			k, v = line.split(":",1)
-			mac = v.strip()
-
-	return mac
-
-debug = open('/tmp/stacki-profile.debug', 'w')
-
-for i in os.environ:
-	debug.write('%s %s\n' % (i, os.environ[i]))	
-
-debug.close()
-
-#
-# make sure the target directory is there
-#
-try:
-	os.makedirs('/tmp/profile')
-except:
-	pass
+with open('/tmp/stacki-profile.debug', 'w') as debug:
+	for i in os.environ:
```

Comment:
This could instead be
```
for key, value in os.environ.items():
    debug.write('%s %s\n' % (key, value))
```

Reply:
Yah, those were like the only few lines I didn't rewrite. Guess I might as well.